<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopIndexDialog</class>
 <widget class="QMainWindow" name="TopIndexDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>522</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>920</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TopIndex</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/toppic.ico</normaloff>:/icon/toppic.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>861</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defaultButton">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outputButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Output Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="outputTextBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>861</width>
      <height>221</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Cantarell';&quot;&gt;Click the Start button to generate index files.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>861</width>
      <height>39</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Database file: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="databaseFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>650</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="databaseFileButton">
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>861</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Fixed modifications:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fixedModComboBox">
       <item>
        <property name="text">
         <string>NONE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Carbamidomethylation on cysteine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Carboxymethylation on cysteine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A fixed modification file</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fixedModFileEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fixedModFileButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_21">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>151</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_21">
     <item>
      <widget class="QCheckBox" name="decoyCheckBox">
       <property name="text">
        <string>Decoy database</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_22">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>331</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Thread number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="threadNumberEdit">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_14">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Mass error tolerance (PPM):</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLineEdit" name="errorToleranceEdit_2">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>15</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>N-terminal forms</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>17</y>
       <width>321</width>
       <height>70</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="NONECheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>NONE</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="NMECheckBox">
        <property name="text">
         <string>NME</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="NMEACCheckBox">
        <property name="text">
         <string>NME_ACETYLATION</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="MACCheckBox">
        <property name="text">
         <string>M_ACETYLATION</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
